e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -293,15 +285,23 @@ test.describe('Notebook entry tests', () => {
 
         expect(await validLink.count()).toBe(1);
     });
-    test.fixme('when an invalid link is entered into a notebook entry, it does not become clickable when viewing', async ({ page }) => {
+    test('when an invalid link is entered into a notebook entry, it does not become clickable when viewing', async ({ page }) => {
         const TEST_LINK = 'www.google.com';
         await page.goto('./#/browse/mine', { waitUntil: 'networkidle' });
 
-        // Create Notebook
-        const notebook = await createDomainObjectWithDefaults(page, {
-            type: 'Notebook',
-            name: "Entry Link Test"
-        });
+        await expandTreePaneItemByName(page, 'My Items');
+
+        await page.goto(notebook.url);
```

Comment:
Since the "mine" folder is not guaranteed to be named "My Items", do this instead

```js
const showSelectedItemButton = page.getByTitle('Show selected item in tree');

// Navigate to the notebook object
await page.goto(notebook.url);

// Reveal the notebook in the tree
await showSelectedItemButton.click();
```

You can repeat that pattern however many times you need.